docs/benchmark/allreduce/report.md

```diff
@@ -0,0 +1,101 @@
+# Benchmark: ElasticDL Calling the Fault-Tolerable AllReduce from Elastic Horovod
+
+We usually use synchronous distributed SGD to train deep learning models in CV
+and NLP. fault tolerance and Elastic scheduling can improve the resource utilization
+and user experience. Horovod has released fault-tolerance AllReduce in v0.20.0.
+ElasticDL implements elastic synchronous training on Kubernetes by calling
+Horovod. So, we experiment to verify the performance of ElasticDL.
+
+## Experiment 1: Elastic Scheduling of Multiple ElasticDL Jobs
+
+Supposed the GPU requirement of two jobs is a little over the total of the
```

Comment:
We will run two model training jobs in the cluster, and the GPU requirement of these jobs exceed the cluster capacity.